### PR TITLE
8282958: Rendering issues of borders, TextFields on Windows High-DPI systems

### PR DESCRIPTION
**Pull request by rajamah**
This fix is based on a similar approach as described here  https://github.com/openjdk/jdk/pull/7449#issuecomment-1068218648.
We rescale the line border to render the Textfield border correctly, so that all sides of the Textfield border are of same thickness.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8282958](https://bugs.openjdk.org/browse/JDK-8282958): Rendering issues of borders, TextFields on Windows High-DPI systems

### Reviewers
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**)
 * [Harshitha Onkar](https://openjdk.org/census#honkar) (@honkar-jdk - Author) ⚠️ Review applies to [0e8fb4e7](https://git.openjdk.org/jdk/pull/10681/files/0e8fb4e742e71bf6bc59263001415e5770345fe3)
 * [Alexander Zuev](https://openjdk.org/census#kizune) (@azuev-java - **Reviewer**)

### Contributors
 * Alexey Ivanov `<aivanov@openjdk.org>`

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/10681/head:pull/10681` \
`$ git checkout pull/10681`

Update a local copy of the PR: \
`$ git checkout pull/10681` \
`$ git pull https://git.openjdk.org/jdk pull/10681/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 10681`

View PR using the GUI difftool: \
`$ git pr show -t 10681`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/10681.diff">https://git.openjdk.org/jdk/pull/10681.diff</a>

</details>

### CODE REVIEW

File: test/jdk/javax/swing/border/LineBorder/ScaledLineBorderTest.java [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+        // Save coordinates of the panels
+        for (Component comp : contentPanel.getComponents()) {
+            panelLocations.add(comp.getLocation());
+        }
+
+        return contentPanel;
+    }
+
+    private static void showFrame(JComponent content) {
+        JFrame frame = new JFrame("Scaled Etched Border Test");
```
Comment (aivanov-jdk) [inline test/jdk/javax/swing/border/LineBorder/ScaledLineBorderTest.java:259]:
```suggestion
        JFrame frame = new JFrame("Scaled Line Border Test");
```

**Review by aivanov-jdk — APPROVED** [review]
Looks good to me.

**Review by aivanov-jdk — APPROVED** [review]

**Review by honkar-jdk — CHANGES_REQUESTED** [review]
Checked the saved images on paint.net, the border thickness is equal and uniform on 4 sides (no. of pixels painted) for different scales.

Copyright year needs to be updated for LineBorder.

**Review by honkar-jdk — APPROVED** [review]

**Review by azuev-java — APPROVED** [review]
Looks good to me.

**Review by aivanov-jdk — APPROVED** [review]

**Pull request closed without merging**